pkg/netpol/eval/eval_test.go

```diff
@@ -1061,7 +1061,8 @@ func setResourcesFromDir(pe *PolicyEngine, path string, netpolLimit ...int) erro
 	var netpols = []*netv1.NetworkPolicy{}
 	var pods = []*v1.Pod{}
 	var ns = []*v1.Namespace{}
-	for _, obj := range objectsList {
+	for i := range objectsList {
+		obj := objectsList[i]
```

Comment:
Why did you do this change here and in other places?

Reply:
to get the linter happier :)
when updated the parser with the new `ANP` object, it raised a linting err called `rangeValCopy` which  detects loops that copy big objects during each iteration; resolved by using index access (as linter suggested)